src/Symfony/Component/Mailer/Test/TransportFactoryTestCase.php

```diff
@@ -37,16 +37,16 @@ abstract class TransportFactoryTestCase extends TestCase
 
     abstract public function getFactory(): TransportFactoryInterface;
 
-    abstract public function supportsProvider(): iterable;
+    abstract public static function supportsProvider(): iterable;
```

Comment:
This looks like a breaking change. The abstract class is extended in all the bridges. At least, the minimum version in composer.json needs to be updated.

https://3v4l.org/g0dGC